lib/trailblazer/operation/controller.rb

```diff
@@ -64,8 +64,9 @@ def operation!(operation_class, params)
       # this is what happens:
       # respond_with Comment::Update::JSON.run(params.merge(comment: request.body.string))
       concept_name = operation_class.model_class.to_s.underscore # this could be renamed to ::concept_class soon.
+      request_body = request.body.respond_to?(:string) ? request.body.string : request.body.read
```

Comment:
Yep, it's not a best approach. However, I leave it this way because you had problems with using just `read` here apotonick/roar-rails@5ae4b28bdb9928ff5685262850d99cb53e81e83f